bundles/org.eclipse.swt/Eclipse SWT Custom Widgets/common/org/eclipse/swt/custom/StyledTextRenderer.java

```diff
@@ -1378,6 +1461,14 @@ void setContent(StyledTextContent content) {
 	maxWidthLineIndex = -1;
 	reset(0, lineCount);
 }
+
+/**
+ * See {@link TextLayout#setFixedLineMetrics}
+ */
+public void setFixedLineMetrics(FontMetrics metrics) {
```

Comment:
Same here, I believe `@since 3.125` is missing

Reply:
OK, in the IDE I see the StyledTextRenderer class is not public, so no since needed here.

Reply:
I guess I could just add it on all public methods to be safe?